frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -1,37 +1,17 @@
-KUBEVIRT_VERSION=$(curl -s https://github.com/kubevirt/kubevirt/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
-#CDI_VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
-CDI_VERSION="v1.29.0"
+#!/bin/bash
 
-VIRTCTL_DOWNLOAD_URL="https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}"
-VIRTCTL_X86_64="${VIRTCTL_DOWNLOAD_URL}-linux-x86_64"
-VIRTCTL_AMD64="${VIRTCTL_DOWNLOAD_URL}-linux-amd64"
+HCO_VERSION="master"
```

Comment:
can we use some version that works for us, o/w we may get stuck with broken master (now called main) versions ?

Reply:
The latest release doesn't install the common templates. I think we should leave it at master/main until they release again.